library/src/main/scala/scala/scalajs/js/typedarray/TypedArray.scala

```diff
@@ -65,6 +65,21 @@ trait TypedArray[T, Repr] extends ArrayBufferView with js.Iterable[T] {
  *  Static information that exists for any concrete TypedArray
  */
 @js.native
-trait TypedArrayStatic extends js.Object {
+trait TypedArrayStatic[T, Repr] extends js.Object {
   val BYTES_PER_ELEMENT: Int = js.native
+
+  /** Returns a new array from a set of elements. */
+  def of(items: T*): Repr = js.native
+
+  /** Creates an array from an `arrayLike` or iterable object. */
+  def from(arrayLike: js.Array[T]): Repr = js.native
+
+  /** Creates an array from an `arrayLike` or iterable object. */
+  def from[E](arrayLike: js.Array[E], mapfn: js.Function1[E, T]): Repr = js.native
+
+  /** Creates an array from an `arrayLike` or iterable object. */
+  def from[E](arrayLike: js.Array[E], mapfn: E =>T): Repr = js.native
```

Comment:
This signature is erroneous. It tries to accept a Scala function, but only JS functions will work. This overload should be removed.